docs/sources/installation/mac.md

```diff
@@ -1,116 +1,306 @@
 
-If you run a container with an exposed port,
+		$ docker hello-world
+		Unable to find image 'hello-world:latest' locally
+		511136ea3c5a: Pull complete 
+		31cbccb51277: Pull complete 
+		e45a5af57b00: Pull complete 
+		hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+		Status: Downloaded newer image for hello-world:latest
+		Hello from Docker.
+		This message shows that your installation appears to be working correctly.
 
-    $ docker run --rm -i -t -p 80:80 nginx
+		To generate this message, Docker took the following steps:
+		 1. The Docker client contacted the Docker daemon.
+		 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+			(Assuming it was not already locally available.)
+		 3. The Docker daemon created a new container from that image which runs the
+			executable that produces the output you are currently reading.
+		 4. The Docker daemon streamed that output to the Docker client, which sent it
+			to your terminal.
 
-then you should be able to access that Nginx server using the IP address reported by:
+		To try something more ambitious, you can run an Ubuntu container with:
+		 $ docker run -it ubuntu bash
 
-    $ boot2docker ip
+		For more examples and ideas, visit:
+		 http://docs.docker.com/userguide/
 
-Typically, it is 192.168.59.103:2375, but VirtualBox's DHCP implementation might change
-this address in the future.
+	
+A more typical way to start and stop `boot2docker` is using the command line.
 
-# Further details
+### From your command line
 
-If you are curious, the username for the boot2docker default user is `docker` and the
-password is `tcuser`.
+Initialize and run `boot2docker` from the command line, do the following:
 
-The Boot2Docker management tool provides several additional commands for working with the
-VM and Docker:
+1. Create a new Boot2Docker VM.
 
-    $ ./boot2docker
-    Usage: ./boot2docker [<options>]
-    {help|init|up|ssh|save|down|poweroff|reset|restart|config|status|info|ip|delete|download|version} [<args>]
+		$ boo2docker init
 
-Continue with the [User Guide](/userguide/).
+	This creates a new virtual machine. You only need to run this command once.
+
+2. Start the `boot2docker` VM. 
+
+		$ boot2docker start
+
+3. Display the environment variables for the Docker client.
+
+		$ boot2docker shellinit
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/ca.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/cert.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/key.pem
+			export DOCKER_HOST=tcp://192.168.59.103:2376
+			export DOCKER_CERT_PATH=/Users/mary/.boot2docker/certs/boot2docker-vm
+
+4. To set the environment variables in your shell do the following:
+
+		$ $(boot2docker shellinit)
+		
+	You can also set them manually by using the `export` commands `boot2docker`
+	returns.
```

Comment:
please don't. The IP address can change randomly, and we've had many users taking the export lines and pasting them into their bashrc / profile.